docs(ast): fix comment for `ClassElement::r#static`

overlookmotel opened the pull request:
Follow-up after #6731. Make a long comment to remove ambiguity about what this method does for once and for all!

overlookmotel commented:
* **#6771** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6771?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Oct 22, 6:37 AM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

overlookmotel committed:
docs(ast): fix comment for `ClassElement::r#static` (#6771)

Follow-up after #6731. Make a long comment to remove ambiguity about what this method does for once and for all!
